modules/datadict/test/datadictTest.php

```diff
@@ -106,6 +106,7 @@ function testDatadictDoespageLoad()
     function testPageUIs()
     {
         $this->safeGet($this->url . "/datadict/");
+        sleep(3);
```

Comment:
line 38 - 42 change to 
           'Source From'        => '.col-xs-12:nth-child(2) .col-sm-3',
            'Name'               => '.col-xs-12:nth-child(3) .col-sm-3',
            'Source Field'       => '.col-xs-12:nth-child(4) .col-sm-3',
            'Description'        => '.col-xs-12:nth-child(5) .col-sm-3',
            'Description Status' => '.col-xs-12:nth-child(6) .col-sm-3',

Reply:
in this case, the html tags has been changed. 
sleep function is not necessary.

Reply:
I'll try that if it doesn't work after rebasing, but I don't see why the HTML tags would have changed. I didn't do anything that should have affected the filters.

Reply:
@kongtiaowang I made your changes and rebased but it still seems to be failing.

Reply:
I will look into it.